test/integration/content-picker/ContentPicker.e2e.test.js

```diff
@@ -34,32 +34,142 @@ const helpers = {
 
         return cy.get('@row');
     },
+    getPaginationArrows: () => {
+        return cy
+            .getByTestId('content-picker')
+            .find('.bdl-Pagination-nav')
```

Comment:
Consider using user-facing text or `data-testid` attributes instead of CSS class names for selectors.